kolibri/plugins/facility_management/assets/src/views/UserTable.vue

```diff
@@ -81,6 +84,9 @@
               {{ user.username }}
             </span>
           </td>
+          <td v-if="$scopedSlots.info">
+            <slot name="info" :user="user"></slot>
+          </td>
```

Comment:
I have a mental block against scoped slots. I read them over and over, and it's never clear to me what props are being passed into what components - so much indirection and aliasing.

That's obviously my problem and I need to wrap my head around them.

That said, I do question whether they're necessary here? For example we could do just:

```html
<td v-if="user.groups">
  <TruncatedItemList :items="user.groups" />
</td>
```

and be done with it.

`UserTable` shouldn't even exist. It's much easier to make a new, custom `CoreTable` instance in every situation rather than contriving to parameterize some abstract sub-concept of "user tables"...